SignalServiceKit/src/Messages/OWSBatchMessageProcessor.m

```diff
@@ -75,8 +74,8 @@ - (OWSSignalServiceProtosEnvelope *)envelopeProto
 
 #pragma mark - Finder
 
-NSString *const OWSMessageContentJobFinderExtensionName = @"OWSBatchMessageProcessingFinderExtensionName";
-NSString *const OWSMessageContentJobFinderExtensionGroup = @"OWSBatchMessageProcessingFinderExtensionGroup";
+NSString *const OWSMessageContentJobFinderExtensionName = @"OWSMessageContentJobFinderExtensionName";
```

Comment:
This will trigger rebuilding the extension, but it should be pretty quick, just the pending jobs, right?